sharktank/conftest.py

```diff
@@ -133,6 +143,20 @@ def pytest_addoption(parser):
         help="Llama3.1 405b fp8 model path",
     )
 
+    parser.addoption(
+        "--google-t5-v1-1-small-fp32-model-path",
+        type=Path,
+        default="/data/t5/small/google__t5-v1_1-small_fp32.gguf",
```

Comment:
We don't hardcode any llama model/tokenizer paths anymore here. You can pass it as an arg directly to pytest.

Reply:
Why not have a default that allows you to `pytest sharktank/tests` if you have data at default paths?

Reply:
I added a comment.